web/src/people.ts

```diff
@@ -869,7 +869,7 @@ export function sender_info_for_recent_view_row(sender_ids: number[]): SenderInf
     const senders_info = [];
     for (const id of sender_ids) {
         // TODO: Better handling for optional values w/o the assertion.
```

Comment:
This TODO probably can be deleted, right?

Reply:
Hmm, maybe this should be `get_user_by_id_assert_valid`; @sahil839 thoughts on that detail?

Reply:
I cannot understand what optional values does the TODO statement talk about, so not sure about that but we can use `get_user_by_id_assert_valid` here. Though this also works fine as we already create user objects for inaccessible message senders when processing message.